src/ts/components/transaction/Amount.tsx

```diff
@@ -0,0 +1,87 @@
+import * as React from 'react'
+import styled from 'styled-components'
+import { Dropdown } from 'semantic-ui-react'
+
+interface IAmountProps {
+  handleAmountChange: any,
+  handleDigitChange: any
+}
+
+
+export default class Amount extends React.Component<IAmountProps> {
+  render () {
+    return (
+      <div>
+        <Label>Amount</Label>
```

Comment:
the text should be render by i18n service for internalization  e.g t(`AmountLabel`)

Reply:
ok adding that in text.